node/config/config.go

```diff
@@ -92,6 +92,8 @@ type RouterNodeConfig struct {
 	Bundle                  channelconfig.Resources
 	MonitoringListenAddress string
 	MetricsLogInterval      time.Duration
+	// ConfigSeq is the number of the last of config block.
```

Comment:
it is not
it is the sequence of the config blocks
or how many config blocks were there 
this is how we reach it in the consenter:

`func (c *Consensus) VerificationSequence() uint64 {
	return c.Config.Bundle.ConfigtxValidator().Sequence()
}`